packages/components/psammead-caption/src/index.jsx

```diff
@@ -14,19 +14,39 @@ import {
 import { getLongPrimer } from '@bbc/gel-foundations/typography';
 import { C_METAL } from '@bbc/psammead-styles/colours';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-import { getSansRegularItalic } from '@bbc/psammead-styles/font-styles';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
 const Caption = styled.figcaption`
-  ${({ script }) => script && getLongPrimer(script)};
-  ${({ service }) => getSansRegularItalic(service)}
+  ${({ script }) => script && getLongPrimer(script)}
+  ${({ service }) => getSansRegular(service)}
   color: ${C_METAL};
-  padding: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX} 0;
+  padding: 0 ${GEL_MARGIN_BELOW_400PX} 0;
   width: 100%;
+  width: -webkit-calc(100% - ${GEL_SPACING});
+  width: -moz-calc(100% - ${GEL_SPACING});
+  width: -o-calc(100% - ${GEL_SPACING});
+  width: calc(100% - ${GEL_SPACING});
+  ${({ dir }) =>
+    dir === 'rtl'
+      ? `margin: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX} 0 0;
```

Comment:
Can I recommend:
- We use long-hand properties, i.e. `margin-top` and `margin-right` to set values, unless _all_ values are greater than 0? This means that `margin-bottom` and `margin-left` are not explicitly set to 0, making it easier to use the cascade in future.
- Instead of setting multiple lines of CSS in a ternary, you could capture the `rtl` styles in a variable and then use that for extra readability. e.g.:

```jsx
const rtlSpacingStyles = `
  margin-top: ${GEL_SPACING};
  padding-bottom: ${GEL_SPACING};
`;

${({ dir }) => dir === 'rtl' ? rtlSpacingStyles  : ltrSpacingStyles};
```

Reply:
Ah! that certainly does look more readable! I'll do that now and alter the margin and padding to be long-hand. Thanks for all the suggestions!